### PR TITLE
docs(react-sdk): add CallStats docs

### PR DESCRIPTION
**Pull request by MartinCupela**

### CODE REVIEW

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx [hunk 1]
```diff
@@ -3,8 +3,28 @@ id: call-statistics
 title: Call statistics
 ---
 
-:::warning
+import CallStats from '../../assets/03-ui-components/call-stats.png';
```
Comment (szuperaz) [inline packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx:6]: This image is missing from git

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx [hunk 2]
```diff
@@ -3,8 +3,28 @@ id: call-statistics
...
 
-:::
+- **Call Latency** graph that outlines average round trip time (in ms).
+- **Call Performance** section that displays the below listed call statistics
+
+<img src={CallStats} alt="Panel displaying call statistics" width={300}/>
+
+### The call statistics displayed:
+
+- **Region** - the SFU we are connected to
```
Comment (szuperaz) [inline packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx:17]: I think SFU is an internal word, I don't think our integrators would necessarily know what it stand for, I'd just use server here

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx [hunk 3]
```diff
@@ -3,8 +3,28 @@ id: call-statistics
...
-TODO: write about: https://github.com/GetStream/stream-video-js/tree/main/packages/react-sdk/src/components/CallStats
+The SDK's default component to display statistics about a call is called `CallStats`. It displays:
 
-:::
+- **Call Latency** graph that outlines average round trip time (in ms).
+- **Call Performance** section that displays the below listed call statistics
+
+<img src={CallStats} alt="Panel displaying call statistics" width={300}/>
+
+### The call statistics displayed:
```
Comment (szuperaz) [inline packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx:15]: Can you please add the `General usage` and `Customization` parts for consistency?

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx (hunk 1 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx:6]: I have added the image

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx (hunk 3 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx:15]: Added the sections

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx (hunk 2 above)
Comment (MartinCupela) [reply packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx]: replaced the use of SFU with server

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx (hunk 3 above)
Comment (szuperaz) [reply packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx:15]: Thank you

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx (hunk 2 above)
Comment (szuperaz) [reply packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx]: Thanks

**Review by szuperaz — APPROVED** [review]

**Review by oliverlaz — APPROVED** [review]

**Pull request merged**